tests/conftest.py

```diff
@@ -1700,10 +1700,20 @@ def core_dump_and_config_check(duthosts, request):
             else:
                 pre_existing_core_dumps = duthost.shell('ls /var/core/')['stdout'].split()
             duts_data[duthost.hostname]["pre_core_dumps"] = pre_existing_core_dumps
-
-            logger.info("Collecting running config before test on {}".format(duthost.hostname))
-            duts_data[duthost.hostname]["pre_running_config"] = \
-                json.loads(duthost.shell("sonic-cfggen -d --print-data", verbose=False)['stdout'])
+            try:
+                golden_config = duthost.shell("cat /etc/sonic/running_golden_config.json", verbose=False)['stdout']
```

Comment:
The ansible module `stat` is able to test if a file exists.

Reply:
done